### DISCUSSION TITLE
How does the extension publisher know that its domain has been verified?

### DISCUSSION THREAD

**Discussion by imbant** [microsoft/vscode-discussions#1090] [Extension Development QnA] [answered]
Hello everyone, I encountered some problems when publishing the extension:

I applied for domain verification for publisher based on [this document](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher) and configured relevant DNS records. "Your request to verify the domain is submitted to the marketplace team for processing." is displayed on the publisher's details page. The document stated that it would be processed within 5 working days. I submitted this application on February 29, 2024, but the details page still shows the same words so far (March 8, 2024).

How do I know the domain verification has passed? Due to my company policy, I want my extension to be domain verified when publishing it

![image](https://github.com/microsoft/vscode-discussions/assets/17983739/fd11bd86-5d82-48df-b760-735db3852921)

**Comment by burkeholland** [comment]
+ @isidorn

**Reply by imbant** [reply, marked as answer]
I have the extension domain verified via marketplace's help. Thanks very much.

**Reply by Aluisio** [reply]
> I have the extension domain verified via marketplace's help. Thanks very much.

I have the same problem. Can you explain to me how you were able to speak to support or how you were able to verify?

**Reply by imbant** [reply]
By email vsmarketplace@microsoft.com

**Reply by Aluisio** [reply]
thank you very much

**Reply by imbant** [reply]
BTW This email address is displayed in a very inconspicuous position on the official website :(

**Reply by Aluisio** [reply]
I just sent an email asking about the progress of the verification. In your case, did it take many days to respond to the email and confirm the verification?

**Reply by imbant** [reply]
I sent my email on Tuesday. The team respond me on Wednesday, and the request was compeleted (I received the response from the same email) on Friday

**Reply by isidorn** [reply]
fyi @seaniyer

**Reply by seaniyer** [reply]
Hi all, 
We have recently updated the verification [process](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#:~:text=New%20publishers%20will%20need%20a%20consistent%20track%20record%20demonstrating%20these%20for%20at%20least%206%20months) to examine the requests more deeply before issuing the blue checkmark. You should get the initial response within the first five days but the time for final approval may vary case by case. 

@Aluisio, sorry the response has been slower. If you haven't heard back, please reply back on the thread with [vsmarketplace@microsoft.com](mailto:vsmarketplace@microsoft.com) and we'll respond with more details.

**Reply by Aluisio** [reply]
> I sent my email on Tuesday. The team respond me on Wednesday, and the request was compeleted (I received the response from the same email) on Friday 

Thanks for the answer

**Reply by Aluisio** [reply]
> Hi all, 
> 
> We have recently updated the verification [process](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#:~:text=New%20publishers%20will%20need%20a%20consistent%20track%20record%20demonstrating%20these%20for%20at%20least%206%20months) to examine the requests more deeply before issuing the blue checkmark. You should get the initial response within the first five days but the time for final approval may vary case by case. 
> 
> 
> 
> @Aluisio, sorry the response has been slower. If you haven't heard back, please reply back on the thread with [vsmarketplace@microsoft.com](mailto:vsmarketplace@microsoft.com) and we'll respond with more details.
> 
> 
> 
> 
> 
> 
> 
> 

Hi @seaniyer. 
I received the email explaining that my request would be analyzed on the same day I made the request, but I thought the entire process would take 5 business days. At this time I sent a response email asking for updates on the process, and on July 12th I sent an email to vsmarketplace@microsoft.com again as per the conversation above with @imbant, and today I received the response since last email with the text “Hi,

Thank you for contacting Visual Studio Marketplace Support. We received your message, and we will get back to you with a response as quickly as possible.  
 
Primary support team business hours are 9AM - 5PM, weekdays Indian Standard Time. Have any additional details that can help us assist you? Feel free to send it by replying to this email”. 
I will continue waiting for the verification process. 
Thank you very much for responding.

**Reply by seaniyer** [reply]
@Aluisio, I confirmed your domain verification request is active in our queue. Although currently it's taking longer than usual, VS Marketplace Support will follow up with you.

**Reply by Aluisio** [reply]
> @Aluisio, I confirmed your domain verification request is active in our queue. Although currently it's taking longer than usual, VS Marketplace Support will follow up with you.

Ok, thank you very much for checking and for your attention @seaniyer

**Reply by Aluisio** [reply]
@seaniyer Hello. After our conversation, my domain was successfully verified, but today the verification simply disappeared. Can you help me?

**Comment by imbant** [comment]
Hi @isidorn @seaniyer guys.

My domain verification disappeared too. The publisher is Craftland Studio(id: craftLandstudio). I previously sent an email to this address [vsmarketplace@microsoft.com](mailto:vsmarketplace@microsoft.com), but 13 days have passed, and I have not received any response. Could there be an issue with the verification policy or my domain?

Please help me—I would truly appreciate your assistance, as this matter is very important to me. Thank you so much for your time and support!

**Reply by isidorn** [reply]
@seaniyer should be able to help. You can also include me in that email thread inikolic@microsoft.com
Thanks

**Reply by imbant** [reply]
Thanks for your help! I have sent a message to you in the email thread.

**Reply by ufukty** [reply]
@isidorn Hi, I've sent an email to `vsmarketplace@microsoft.com` 2 weeks ago but got no response so far. Since my initial application was months ago I consider now the possibility of rejection without a notice. Should I give up or would it help if I resend the email with your address included? 

I've just added a lot of unique content to my website which was previously only a link tree to my public profiles in social sites after I've suspected the emptiness might be the reason of silence.

**Reply by isidorn** [reply]
@seaniyer should be able to help here

**Reply by seaniyer** [reply]
Hi 

If you still haven't received a response from support, please add me (seiyer@microsoft.com) in the email thread and I'll help follow up.

**Reply by ufukty** [reply]
Still got no response after 4 days. Should I cancel the process and start again or give up? It is either buried under a pile, or there is a silent rejection.

**Reply by seaniyer** [reply]
@ufukty  
I'd like to track down your email to see how we can help. Can you cc me on the email or provide the domain name? Also adding @madhurivadaligithub for help.

**Reply by ufukty** [reply]
My last email 

From: ufukty@outlook.com
To: vsmarketplace@microsoft.com
Cc: seiyer@microsoft.com
Date: Tue, Jun 24, 2025, 06:31

My first email

From: ufukty@outlook.com
To: vsmarketplace@microsoft.com
Date: Sat, Jun 7, 2025, 09:15

UTC+03

**Reply by ufukty** [reply]
Email dates are in UTC+03

**Reply by v-manil2** [reply]
Hi @ufukty Apologies for the delay/inconvenience caused. we have successfully verified your domain, Kindly review and confirm.

**Reply by ufukty** [reply]
@seaniyer @v-manil2 @isidorn Thanks for your time anyone involved. I can see the verification mark on both the management website and the marketplace extension page. I expect the VSCode will update its cache with some delay. Maybe tomorrow.

Is there any reason you are allowed to share on why it took this long? Was it about the lack of some content in my website? Additional information might be helpful for future visitors.

**Reply by seaniyer** [reply]
@imbant, I believe your verification request has also been done. Let us know if you need further help.

**Reply by imbant** [reply]
Hi, my request was completed a few days after this comment. Thanks for your help!
